mmv1/third_party/terraform/services/compute/resource_compute_router_nat_test.go.tmpl

```diff
@@ -804,6 +804,40 @@ func testAccCheckComputeRouterNatDelete(t *testing.T, n string) resource.TestChe
 	}
 }
 
+{{ if ne $.TargetVersionName `ga` -}}
+func TestAccComputeRouterNat_withNat64Configuration(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
```

Comment:
```suggestion
		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
```

& remove `google-beta` from test configs- the implicit `google` provider will be the beta provider inside these tests. We only use the specific beta factory for example tests so the docs use the beta provider!